sdk/core/azure-core/inc/azure/core/http/win_http_transport.hpp

```diff
@@ -4,6 +4,7 @@
 /**
  * @file
  * @brief #Azure::Core::Http::HttpTransport implementation via WinHTTP.
+ * cspell:words HCERTIFICATECHAIN PCCERT CCERT HCERTCHAINENGINE HCERTSTORE
  */
```

Comment:
This comment will get included in doxygen documentation (it is in `/** ... */` section. It needs to go in a separate comment section.
```suggestion
 */
// cspell:words HCERTIFICATECHAIN PCCERT CCERT HCERTCHAINENGINE HCERTSTORE
```